client/wfnews-war/src/main/angular/src/app/components/dashboard-component/widgets/summary-widget/summary-widget.component.html

```diff
@@ -9,7 +9,7 @@
       </div>
     </div>
     <div class="stat-box">
-      <img height="36" width="36" src="/assets/images/svg-icons/warning.svg" alt="Order" class="stat-box-icon" />
+      <img height="36" width="36" src="/assets/images/svg-icons/add.svg" alt="Order" class="stat-box-icon" />
```

Comment:
we might want to double check the 'alt' attributes. (keep it same with icon or image name)  I remember we used to enfore these for accesibility reason in public mobile, and I assume someone will use screen reader in news app too